ateam_ui/src/src/components/StatusComponent.vue

```diff
@@ -144,12 +144,12 @@ export default {
 
             // Theres probably a better way to do this
             const wheels = {
-                startX: [-.075, .075, -.035, .035, -.05],
-                startY: [-.095, -.095, .12, .12, -.09],
-                endX: [-.12, .12, -.1, .1, .05],
-                endY: [-.03, -.03, .07, .07, -.09],
-                textX: [-.12, .12, -.12, .12, 0],
-                textY: [-.12, -.12, .14, .14, -.13]
+                startX: [-.075, -.035, .035, .075, -.05],
+                startY: [-.095, .12, .12, -.095, -.09],
+                endX: [-.12, -.1, .1, .12, .05],
+                endY: [-.03, .07, .07, -.03, -.09],
+                textX: [-.12, -.12, .12, .12, 0],
+                textY: [-.12, .14, .14, -.12, -.13]
             }
 
             // TODO: Figure out what order the motor numbers use
```

Comment:
0: FL
1: BL
2: BR
3: FR
4: DRB

Reply:
The wheels dict appears to have this order correct, so I'm guessing this TODO is just leftover and I'm going to remove it.

Reply:
The wheels struct above seems to have them in the right order, so I'm guessing this TODO is leftover and I'll remove it.